cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -381,6 +382,21 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 		}
 	}
 
+	groupVersion = "batch/v2alpha1"
```

Comment:
Did you mean we should use something like `batchapiv2alpha1.SchemeGroupVersion` instead?